layers.go

```diff
@@ -1924,7 +1933,6 @@ func (r *layerStore) Modified() (bool, error) {
 	}
 	if info != nil {
```

Comment:
If (per the above) this is supposed to guard against manual modifications without holding any locks …